__mocks__/LocalizeMock.js

```diff
@@ -5,13 +5,15 @@ const Localize = ({ i18n_default_text, values }) => {
     return localizedText || null;
 };
 
+const mockFn = jest.fn((text, args) => {
+    return text.replace(/{{(.*?)}}/g, (_, match) => args[match.trim()]);
+});
+
 // Mock for useTranslations hook
 const useTranslations = () => ({
-    localize: jest.fn((text, args) => {
-        return text.replace(/{{(.*?)}}/g, (_, match) => args[match.trim()]);
-    }),
+    localize: mockFn,
```

Comment:
why move the function out? 👀

Reply:
didn't move it out.. just replaced it with the reusable mock function.